package.json

```diff
@@ -38,7 +39,7 @@
     "test": "mocha",
     "lint": "eslint \"**/*.js\" Jakefile",
     "coverage": "istanbul cover node_modules/mocha/bin/_mocha",
-    "doc": "rimraf out && jsdoc -c jsdoc.json lib/* docs/jsdoc/*",
+    "doc": "jake doc",
     "devdoc": "rimraf out && jsdoc -p -c jsdoc.json lib/* docs/jsdoc/*"
```

Comment:
Makes sense -- although I don't actually envision anybody doing this task on Windows. :) Also, might make more sense to use a parameterized Jake task for this: `jake doc[dev]`: http://jakejs.com/docs#tasks_passing_parameters_to_jake